components/com_finder/src/Model/SuggestionsModel.php

```diff
@@ -109,7 +109,7 @@ protected function getListQuery()
             ->where('l.access IN (' . implode(',', $groups) . ')')
             ->where('l.state = 1')
             ->where('l.published = 1');
-        $aQuery->select('DISTINCT o.term FROM (' . $termQuery . ') AS o');
+        $aQuery->select('DISTINCT(o.term)')->from('(' . $termQuery . ') AS o');
```

Comment:
```suggestion
        $aQuery->select('DISTINCT o.term')->from('(' . $termQuery . ') AS o');
```
Please change back to the right officially documented syntax `SELECT DISTINCT columnname` instead of the undocumented `SELECT DISTINCT(columnname)`.